### PR TITLE
Contact form section

### PR DESCRIPTION
**Pull request by christianbiring1**
In this milestone I had to :

- [ ] Add a contact form to my portfolio and,
- [ ] Make sure that I guide my users' experience in a way that helps them fill in the form without frustration.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -246,22 +246,36 @@ <h1>contact me</h1>
...
+            required
+          /><br />
+          <input
+            type="email"
+            id="email"
+            name="mail"
+            placeholder="christianbiringanine22@gmail"
+            required
+          /><br />
           <textarea
```
Comment (mikethreels) [inline index.html]: - [x] please don't use your own name as placeholder text, what is intended with placeholder text is for example "full name" in the name input and "email address" in the mail input field.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Christian, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (christianbiring1) [reply index.html]: Alright, @mikethreels I got you. let me change that real quick.

**Review by Deepakdanger — APPROVED** [review]
Hi @christianbiring1,

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**